gettsim/transfers/kinderzuschl/kost_unterk.py

```diff
@@ -103,56 +84,25 @@ def _kinderzuschl_wohnbedarf_eltern_anteil(
     -------
 
     """
-    kinder_in_tu = tu_id.replace(anz_kinder_tu)
-    erwachsene_in_tu = tu_id.replace(anz_erwachsene_tu)
-    conditions = []
-    choices = []
     ex_min = kinderzuschl_params["exmin"]
```

Comment:
```suggestion
    ex_min = kinderzuschl_params["existenzminimum"]
```

No need to abbreviate (and use two different abbreviations)